docs/providers/aws/guide/README.md

```diff
@@ -17,5 +17,3 @@ Welcome to the Serverless AWS Lambda Guide!
 Get started with the **[Introduction to the framework](./intro.md)**
 
 If you have questions, join the [chat in gitter](https://gitter.im/serverless/serverless) or [post over on the forums](https://forum.serverless.com/)
```

Comment:
We don't link out to gitter from our site anymore. We should replace this with a link to Slack

Reply:
The Slack link is https://serverless.com/slack